h5p.classes.php

```diff
@@ -2130,6 +2134,26 @@ public function getDependenciesFiles($dependencies, $prefix = '') {
     return $files;
   }
 
+    public function hasPresave($libraryName)
+    {
+        if( isset($this->h5pD) ){
+            extract(H5PCore::libraryFromString($libraryName));
```

Comment:
Please don't use extract, it makes it harder to follow the code and may expose unintended variables as well as making static code analysis much harder. Especially when the function the values are retrieved from is not properly documented.